components/net/image_cache_thread.rs

```diff
@@ -551,18 +544,13 @@ impl ImageCache {
                                 FetchResponseMsg::ProcessRequestBody |
                                 FetchResponseMsg::ProcessRequestEOF => return,
                                 FetchResponseMsg::ProcessResponse(meta_result) => {
-                                    ResponseAction::HeadersAvailable(meta_result.map(|m| {
-                                        match m {
-                                            FetchMetadata::Unfiltered(m) => m,
-                                            FetchMetadata::Filtered { unsafe_, .. } => unsafe_
-                                        }
-                                    }))
+                                    FetchResponseMsg::ProcessResponse(meta_result)
                                 }
                                 FetchResponseMsg::ProcessResponseChunk(new_bytes) => {
-                                    ResponseAction::DataAvailable(new_bytes)
+                                    FetchResponseMsg::ProcessResponseChunk(new_bytes)
                                 }
                                 FetchResponseMsg::ProcessResponseEOF(response) => {
-                                    ResponseAction::ResponseComplete(response)
+                                    FetchResponseMsg::ProcessResponseEOF(response)
```

Comment:
No - if you commit the changes and push to the same branch, this pull request will be updated. Go ahead and use `git commit --amend` to merge the changes into the same commit, or [squash](https://ariejan.net/2011/07/05/git-squash-your-latests-commits-into-one/) if there are already multiple commits.